### ISSUE TITLE
delta delta ct calculation

### ISSUE THREAD

**Issue by mkendzel** [mkendzel/Auto-qPCR#1]
On test set, the delta delta ct values were about 2% off from those calculated manually. It seems that the test data set has some odd organization. The error could come from a miss selection of rows, or it could come from the data formatting. Going to get a new data set and test with that.

**Comment by mkendzel** [comment]
The training data set's format was the initial issue. The calculation was correct and was confirmed with a new data set
